pkgs/development/compilers/august/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "august";
+  version = "unstable-2023-08-13";
+
+  src = fetchFromGitHub {
+    owner = "yoav-lavi";
+    repo = "august";
+    rev = "42b8a1bf5ca079aca1769d92315f70b193a9cd4a";
+    hash = "sha256-58DZMoRH9PBbM4sok/XbUcwSXBeqUAmFZpffdMKQ+dE=";
+  };
+
+  cargoHash = "sha256-/GvBbsSL0dZ0xTystIpb8sk1nNg5hmP4yceCHlh7EQE=";
+
+  postInstall = ''
+    mv $out/bin/{august-cli,ag}
```

Comment:
Why would you do this?

Reply:
This is what upstream documented: https://github.com/yoav-lavi/august#cli-usage-cli-command-tbd

> ```
> ag -o / --output [OUTPUT] (json|toml|yaml) [INPUT]
> ```

Reply:
hm, okay. i suppose its fine then.
I am not sure what the policy on binary name clashes is...